extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/tunnel/AbstractGuacamoleTunnelService.java

```diff
@@ -201,87 +201,54 @@ protected abstract void release(RemoteAuthenticatedUser user,
             ModeledConnectionGroup connectionGroup);
 
     /**
-     * Returns a guacamole configuration containing the protocol and parameters
-     * from the given connection. If the ID of an active connection is
-     * provided, that connection will be joined instead of starting a new
-     * primary connection. If tokens are used in the connection parameter
-     * values, credentials from the given user will be substituted
-     * appropriately.
-     *
-     * @param user
-     *     The user whose credentials should be used if necessary.
+     * Returns a GuacamoleConfiguration which connects to the given connection.
+     * If the connection ID of an active is provided, that active connection
+     * will be joined rather than establishing an entirely new connection. If
+     * a sharing profile is provided, the parameters associated with that
+     * sharing profile will be used to define the access provided to the user
+     * accessing the shared connection. If tokens are used in the connection
+     * parameter values of the sharing profile, credentials from the given user
+     * will be substituted appropriately.
```

Comment:
Right - that wording is wrong as of commit 1210d5624c4eb173417cab8358eca4cc3b6c0ebe (which deprecated the `StandardTokens` class). This function used to perform token substitution, but no longer needs to and hasn't since that commit.

I've now removed that "If tokens are used ..." sentence.